templates/website/styles/styles.scss

```diff
@@ -56,4 +72,16 @@ $lg: bsiProperty('gridBreakpointLG', 940px);
     margin-left: auto;
     margin-right: 0;
   }
+}
+
+// content editor specific CSS
+.bsi-ce-edit-mode .nav-item.dropdown.language-switch {
+  &:hover {
+    .dropdown-menu {
+      display: none;
+    }
+  }
+  .active-language {
+    text-transform: uppercase;
```

Comment:
das `text-transform: uppercase;` könntest du gleich allgemeingültig machen und auch die anderen Einträge in der Liste mit CSS uppercasen...